themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -7,6 +7,12 @@
 
   // More link should use default limit, not custom limit:
   $moreUrl = $this->url($params->getOptions()->getSearchAction()) . $results->getUrlQuery()->setPage(1)->setLimit($params->getOptions()->getDefaultLimit());
+
+  // Advanced search link
+  $advancedSearchAction = $params->getOptions()->getAdvancedSearchAction();
+  $advancedSearchUrl = $advancedSearchAction
+    ? $this->url($advancedSearchAction, [], ['query' => ['edit' => $results->getSearchId()]])
+    : false;
```

Comment:
This code is adapted from searchbox.phtml.  In that file there is more complicated logic for creating the URL based on $hiddenFilters if no results are present.  I don't think that's needed here but I may be wrong.

Reply:
@EreMaijala might have the strongest opinion on this point.

Reply:
I think that search id should be enough, but this needs to be tested with SearchTabs and SearchTabsFilters.

Reply:
> I think that search id should be enough, but this needs to be tested with SearchTabs and SearchTabsFilters.

Ok I tested this.  The code works fine with SearchTabs in use or not.  It also works when hidden filters are in place for the combined sections; the advanced link contains the search ID that somehow has saved the hidden filter behind the scenes.  Honestly I thought this was broken for a while before I saw that it was working -- I haven't worked with hidden filters before.  Kudos for such a cool capability!